README.md

```diff
@@ -1 +1,167 @@
+- [📝 License](#license)
+
+# 📖 Capstone 1 <a name="about-project"></a>
+
+**Capstone 1** is part of introductory projects in Microverse software development program.
+The template adapted for the project is for an event program. Some of its contents were redacted as part of the provided guidlines.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- HTML
+- CSS
+- Javascript
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="">N/A</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="">N/A</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="">N/A</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Build an event website**
+- **Dynamic loading of website contents**
+- **Click controls of website elements**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link - N/A]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+In order to run this project you need Live Server for your local testing
+
+### Setup
+
+Clone this repository to your desired folder on your local machine
+
+### Install
+
+Install this project with:
+
+### Usage
+
+To run the project, execute the following command:
+
+N/A
+
+### Run tests
+
+To run tests, run the following command:
+
+N/A
+
+### Deployment
+
+You can deploy this project using:
+
+[Github Page] (https://pages.github.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Damilare Adepoju**
+
+- GitHub: [@githubhandle](https://github.com/adamilare)
+- Twitter: [@twitterhandle](https://twitter.com/mailtodare)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/adamilare)
+- GitUp Page: [Capstone-1](https://adamilare.github.io/Capstone-1/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- Implement Registration Form
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/adamilare/Capstone-1/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please give it a thumb up.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank my fellow coding partners;
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] As you have used someone else design, it is important to give a credit to him/her as a privilege. This helps anyone who visits your website but is interested in the design you have followed get to know the source 💡. Moreover, it is among the project requirements to give credit to [Cindy Shin](https://www.behance.net/adagio07) the author of the original design you used. Please add her here in the Acknowledgement section and the link to the design. 😃 👍